app/precompiled.py

```diff
@@ -138,37 +173,39 @@ def sanitise_file_contents(encoded_string):
         }
 
 
-@precompiled_blueprint.route("/precompiled/overlay.<file_type>", methods=['POST'])
+@precompiled_blueprint.route("/precompiled/overlay.png", methods=['POST'])
```

Comment:
We do call this function with a filetype of `pdf`, so I think it will still have to work for both filetypes.

Admin calls api here https://github.com/alphagov/notifications-admin/blob/a49b492ade6f658c438f724728ef940a4221bf3a/app/main/views/notifications.py#L60-L62 which, then makes the request to template preview with a filetype of `pdf`